rabbitmq/content.md

```diff
@@ -54,6 +54,26 @@ RABBITMQ_SSL_VERIFY
 RABBITMQ_VM_MEMORY_HIGH_WATERMARK
 ```
 
+### Setting default user and password
+
+If you wish to change the default username and password of `guest` / `guest`, you can do so with the `RABBITMQ_DEFAULT_USER` and `RABBITMQ_DEFAULT_PASS` environmental variables:
```

Comment:
I'd like to make it very clear that these are not docker-specific variables. So something like this:
```suggestion
If you wish to change the default username and password of `guest` / `guest`, you can do so with the `RABBITMQ_DEFAULT_USER` and `RABBITMQ_DEFAULT_PASS` environmental variables. These variables were available previously in the docker-specific entrypoint shell script but are now available in RabbitMQ directly.
```
Or maybe something shorter?
```suggestion
If you wish to change the default username and password of `guest` / `guest`, you can do so with the `RABBITMQ_DEFAULT_USER` and `RABBITMQ_DEFAULT_PASS` upstream-supported environmental variables:
```
What do you think @tianon?

Reply:
I like the much-more-explicit first version :sweat_smile: